src/redturtle/chefcookie/browser/cookie_config.py

```diff
@@ -354,32 +353,24 @@ def get_tech_cookies_config(self):
         }
 
         labels = self.get_registry_settings(name="technical_cookies_labels")
-        analytics_cookies_labels = self.get_registry_settings(
-            name="analytics_cookies_labels"
-        )
-        matomo_cookies_labels = self.get_registry_settings(name="matomo_cookies_labels")
-        functional_cookies_labels = self.get_registry_settings(
-            name="functional_cookies_labels"
+        specific_labels = self.get_registry_settings(
```

Comment:
perché specific_label come nome? in generale perché li ha chiamati *specific*

Reply:
boh, era il primo nome che mi è venuto in mente